css/build.css

```diff
@@ -0,0 +1,419 @@
+body {
+  padding-bottom: 0 !important;
+}
+
+[data-interactive-floorplan-id] {
+  opacity: 1;
+  transition: opacity 0.25s ease-out;
+}
+
+[data-interactive-floorplan-id].is-loading {
+  opacity: 0;
+  pointer-events: none;
+}
+
+.flooplan-app-placeholder {
+  padding: 20px;
+  text-align: center;
+  background: rgba(51,51,51,0.1);
+  border: 4px solid rgba(51,51,51,0.15);
+}
+
+.floorplan-component-wrapper {
+  position: relative;
+  height: 100vh;
+}
+
+body.fl-with-top-menu .floorplan-component-wrapper {
+  height: calc(100vh - 44px);
+  height: calc(100vh - (44px + env(safe-area-inset-bottom) + env(safe-area-inset-top)));
+}
+
+body.fl-with-bottom-menu .floorplan-component-wrapper {
+  height: calc(100vh - 65px);
+  height: calc(100vh - (65px + env(safe-area-inset-bottom) + env(safe-area-inset-top)));
+}
```

Comment:
What about `[data-has-notch]` to make sure it's correctly supported and rendered in Studio with the iPhone XS family device?